SocketTimeoutException when ES is initialized.

I get the following error when the plugin initializes ES via init script:

`[ERROR] Failed to execute goal com.github.alexcojocaru:elasticsearch-maven-plugin:5.1:runforked (start-elasticsearch) on project es: Execution start-elasticsearch of goal com.github.alexcojocaru:elasticsearch-maven-plugin:5.1:runforked failed: Cannot execute command PUT:test_index:{}: java.net.SocketTimeoutException: Read timed out -> [Help 1]`

Sorry for the delay.

Plugin output:
`
[INFO] --- elasticsearch-maven-plugin:5.3-SNAPSHOT:runforked (start-elasticsearch) @ project-rest ---
[INFO] Using Elasticsearch [0] configuration: com.github.alexcojocaru.mojo.elasticsearch.v2.InstanceConfiguration@51b12aa8[id=0,baseDir=D:\work\COMPANY\project\sites-project\project-rest\target\elasticsearch0,httpPort=9200,transportPort=9300,pathData=<null>,pathLogs=<null>]
[DEBUG] Resolving ElasticsearchArtifact[org.elasticsearch.distribution.zip:elasticsearch:zip:5.1.1]
[INFO] Starting Elasticsearch [0] in directory 'D:\work\COMPANY\project\sites-project\project-rest\target\elasticsearch0' with arguments 'cmd /c .\bin\elasticsearch -p pid -Ecluster.name=localcluster -Ehttp.port=9200 -Etransport.tcp.port=9300 -Eaction.auto_create_index=false -Ehttp.cors.enabled=true'
[DEBUG] Waiting  up to 60s for Elasticsearch to start ...
[DEBUG] Sending GET request to http://localhost:9200/
[DEBUG] Sending GET request to http://localhost:9200/
[DEBUG] Sending GET request to http://localhost:9200/
[DEBUG] Response with status code 200 and content: {  "name" : "bUOa2FY",  "cluster_name" : "localcluster",  "cluster_uuid" : "v33mr-y3R3WXveQNb3hCAw",  "version" : {    "number" : "5.1.1",    "build_hash" : "5395e21",    "build_date" : "2016-12-06T12:36:15.409Z",    "build_snapshot" : false,    "lucene_version" : "6.3.0"  },  "tagline" : "You Know, for Search"}
[DEBUG] Elasticsearch has started
[DEBUG] Parsing command: # create the index
[DEBUG] Parsing command: PUT:test-index:{}
[INFO] Sending PUT request to http://localhost:9200/test-index with entity '{}'
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 36.845 s
[INFO] Finished at: 2017-01-04T10:10:50+03:00
[INFO] Final Memory: 109M/1327M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.alexcojocaru:elasticsearch-maven-plugin:5.3-SNAPSHOT:runforked (start-elasticsearch) on project project-rest: Execution start-elasticsearch of goal com.github.alexcojocaru:elasticsearch-maven-plugin:5.3-SNAPSHOT:runforked failed: Cannot execute command PUT:lease-deals:{}: java.net.SocketTimeoutException: Read timed out -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
[INFO] Stopping the Elasticsearch process at application shutdown ...
[INFO] ... the Elasticsearch process has stopped. Exit code: 1
[INFO] Elasticsearch [0] stopped with exit code 1`

--- a/src/main/java/com/github/alexcojocaru/mojo/elasticsearch/v2/ForkedInstance.java
+++ b/src/main/java/com/github/alexcojocaru/mojo/elasticsearch/v2/ForkedInstance.java
@@ -48,7 +48,7 @@ public class ForkedInstance
 
         ProcessBuilder processBuilder = new ProcessBuilder(getStartScriptCommand());
         processBuilder.directory(baseDir);
-        processBuilder.redirectErrorStream(true);
+        processBuilder.inheritIO();
         
         Log log = config.getClusterConfiguration().getLog();
 
@@ -90,7 +90,7 @@ public class ForkedInstance
 
         ProcessBuilder processBuilder = new ProcessBuilder("chmod", "755", "elasticsearch");
         processBuilder.directory(binDirectory);
-        processBuilder.redirectErrorStream(true);
+        processBuilder.inheritIO();
 
         Log log = config.getClusterConfiguration().getLog();
 

